forge-gui/res/cardsfolder/upcoming/treacherous_greed.txt

```diff
@@ -0,0 +1,8 @@
+Name:Treacherous Greed
+ManaCost:1 W B
+Types:Instant
+A:SP$ Draw | Cost$ B Sac<1/Creature.dealtDamageThisTurn/creature that was damaged this turn> | NumCards$ 3 | SubAbility$ DBLoseLife | SpellDescription$ Draw three cards.
```

Comment:
`/creature that was damaged this turn>` bad description here

Careful, cost needs to line up with ManaCost better